.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - 3.6
+before_script:
+  - cp custom.conf.template custom.conf
```

Comment:
What's the purpose of this line?

Reply:
The code in `config.py` gets run, but if there is no real `custom.conf` file it throws a (valid) exception. See here: https://github.com/dimagi/required-labels/pull/3/files#diff-fdf313ed2303059c44a9ebfffbef8eebR20